src/tests/ftest/pool/create_query.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2021-2022 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from pool_test_base import PoolTestBase
+
+
+class PoolCreateQueryTests(PoolTestBase):
+    # pylint: disable=too-many-ancestors
+    """Pool create tests.
+
+    All of the tests verify pool create performance with 7 servers and 1 client.
+    Each server should be configured with full compliment of NVDIMMs and SSDs.
```

Comment:
Is there a test tag for NVMe drives and unsimulated pmem? I don't know if "hw" always implies real storage.

Reply:
There is a tag `nvme`, but not a similar `pmem` as far as I can tell. And `hw` just means it runs on a hardware cluster instead of VM. Hardware clusters always have real hardware, but the test could theoretically not use it.

Reply:
Seems like this test should at least tag for nvme then. So by real hardware you mean all the CI nodes have physical NVDIMMs? As stated here, that seems to be a requirement for this test.

Reply:
I confirm, that having PMEM and and NVMe is needed for this test as we want to check that the size allocated on these devices are consistent with the input parameters.

Reply:
Superficially skimming through it, it looks like NVMe nodes generally have physical PMEM. So the nvme tag may be sufficient for now.